app/views/users/list.html.haml

```diff
@@ -1,14 +0,0 @@
-%h1 Users
```

Comment:
This should have become index.html.haml as it LISTS all the users.  Index => LIST, show => GET ONE.